assets/src/components/cd/clusters/ClusterUpgradeFlyover.tsx

```diff
@@ -461,7 +467,10 @@ function ClusterUpgradeAccordionTrigger({
               color="icon-success"
             />
           ) : (
-            <Icon width={16} />
+            <Icon
+              color="icon-warning"
+              width={16}
```

Comment:
same, should be a theme value

Reply:
IIRC 16px is default value, so it might be possible to skip it.